modules/arch-openshift-ansible-broker.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * architecture/service-catalog.adoc
+
+[id='openshift-ansible-broker-{context}']
+= OpenShift Ansible Broker
+
+The OpenShift Ansible broker (OAB) is an implementation of the Open Service
```

Comment:
@kalexand-rh I'm pulling in this first paragraph as sb-about-ansible-service-broker.adoc with this PR [1] in case you want to eventually rebase and adjust to include this file instead. Should be merged in the next day or so.

I didn't really want all of the extra detail included in this file, but I could probably be convinced to include it so that we can just easily share the module between our two places :).

[1] https://github.com/openshift/openshift-docs/pull/14067

Reply:
I'd rather reuse the same module. If you don't think that info is useful anywhere, we might be able to drop it.

Reply:
@kalexand-rh You've convinced me :) I included the rest of the content in this file (with slight modifications), so you can just reuse that module.

CC @chrisnegus since @vikram-redhat mentioned that you were going to be working on the arch section - that I'm adding 3 modules that can be reused (see my other comments)